kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/CompactCommand.java

```diff
@@ -43,6 +43,10 @@ public CompactCommand(Logger console) {
       description="The number of writer processes to use")
   int numWriters = -1;
 
+  @Parameter(names={"--num-partition-writers"},
```

Comment:
I'd like to make this a bit easier to understand. What about `--files-per-partition`?